examples/push-script.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+#script takes first param as current Voyager release version and next params the names of directories of examples that should be pushed
```

Comment:
Instead of taking the current release version as an argument, it would be nicer to take it as an environment variable. See this example here: https://github.com/aerogear/voyager-server/blob/master/scripts/validateRelease.sh#L3-L4

This keeps the way in which we invoke our bash scripts consistent which is especially important for CI related stuff. It's also a bit more readable to have nicely named variables IMO

The end result will be instead of calling the script with `./push-script.sh $package_version basic` we will do `TAG=$package_version ./push-script.sh basic`

Reply:
ok